packages/x-date-pickers/src/internals/components/PickerProvider.tsx

```diff
@@ -118,6 +121,11 @@ export interface PickerContextValue<
    * Is always equal to "portrait" if the component you are accessing the context from is not wrapped by a picker.
    */
   orientation: PickerOrientation;
+  /**
+   * Whether the heavy animations should be disabled.
+   * @default `@media(prefers-reduced-motion: reduce)` || `navigator.userAgent` matches Android <10 or iOS <13
+   */
+  reduceAnimations?: boolean;
```

Comment:
Not sure if we should name this one with a boolean prefix (`shouldReduceAnimations` for instance) or not.
I kept the name of the prop for now.

I added it to the public context because I think it's something people should be able to use, especially for custom views when we will stop drilling it to the view.